zeitgeistpm/ui
# Promoted markets

**Work done:**

- [x] Implement CMS promoted markets data.
- [x] Implement promoted markets data fetching from notion CMS.
- [x] Implement promotion callout + modal on market page.
- [x] Determine if we show modal automatically on page load or have a clickable ui element on the market page that opens it. @mk26071980 @lsaether 
- [ ] Check with marketing if CMS data structure fits requirements. @lsaether @JamesPrestonZA 
- [x] Slight improvement to modal enter and leave animation.

**Testing**
1. Visit a promoted market, like [(URL).](https://ui-git-750-promoted-markets-zeitgeistpm.vercel.app/markets/190)
2. See that the promoted market modal appears since you have not visited it yet.
3. Close the popup.
4. On refresh the popup should not show.
5. Click the "Promo Market Rules" under the market image and see that the popup reappears.
6. Play with the CMS data and see that it reflects the changes(not sure if caching prevents immediate updates)

Fixes #750
